### ISSUE TITLE
documentation questions, regarging use on JVM, JS and Native

### ISSUE THREAD

**Issue by bblfish** [cquiroz/scala-java-time#464]
I was re-implementing a a.pure scala version of banana-rdf, which needs xsd dateTimestamps, and then I realised I need a pure Scala implementation for scalajs. and following links I found your project.

Your documentation states:
> The scala-java-time library is currently available for Scala (JVM, version 8 and later) and Scala.js (JavaScript).

Once I noticed that you use the same `java.time` namespace I got a bit confused, because I wondered if one was meant to override the jvm package, and how one was meant to do that. But it seems one should not use this lib for the jvm.

So I guess this is primarily useful for scala-js and Scala-native, right? (Which is actually what I was looking for)

If so that could be made a bit clearer on the documentation page.

**Comment by cquiroz** [comment]
That's right, the main reason for the jvm side is so we can do tests comparing the results to the runtime behavior of the jwm.

In reality you only want to use it on js/native.

we should update the documentation, PRs are welcome
